### DISCUSSION TITLE
大家有戴手錶的習慣嗎

### DISCUSSION THREAD

**Discussion by yuchien-lin** [1101-GS4719-C/Python-Community-GS4719-C#1899] [General]
我從國三開始就不習慣戴手錶，因為當時要會考不能戴電子錶，當時戴的鐘錶不我戴在手上不太舒服，
所以之後考試戴的鐘錶我都是放鉛筆盒裡，自此我就不習慣戴手錶在手上了，要看時間都拿手機看，
所以我想問一下，是否大家現在有戴手錶的習慣?

**Comment by BeaGle0928** [comment]
考試的時候肯定是要戴手錶的，但隨著年紀增長，有時候戴手錶好像也不是真的為了要看時間？可能對於部分的人而言，手錶跟帽子手環一樣只是裝飾吧。

**Reply by woohsin** [reply]
因為現在都用手機看時間了吧

**Comment by sorn32** [comment]
我現在是只有考試才會特別戴手錶，一般戴在手上都會覺得卡卡的，也很怕不小心撞到，而且現在手機真的很方便，就更少戴了(´･_･`)

**Comment by chuanmin0317** [comment]
在小時候，為了看時間，我一直都有戴著手表的習慣，不過隨著年紀的增長，手錶對我來說比較像是裝飾的作用，因為以前戴的那隻錶不太喜歡，等手錶沒電之後就沒戴在手上了，之後也沒再戴手錶了

**Reply by woohsin** [reply]
我是有一次考試需要才發現我用很久的那隻手表已經沒電了

**Comment by vchien03** [comment]
大學以前，我只有在考試時會戴手錶，其他時候都是看教室時鐘或手機。不過上大學之後，因為想監測自己的心跳跟睡眠狀況而開始使用小米手環，不知不覺就養成了用手表看時間的習慣，有幾次沒有戴手錶反而會覺得不習慣~

**Reply by woohsin** [reply]
我現在連考試都沒甚麼戴手錶了

**Comment by benbebobo149** [comment]
我自己很不喜歡戴手錶，因為我是個非常喜歡球類運動的人。戴手錶除了會限制你的表現，更重要的是，可能會造成其他人受傷。我曾在打籃球時被其他人的手錶割傷過，這讓我對戴手錶的印象更加負面。另外，有時候脫下手錶時可能會忘記拿走，或者被人偷走，這讓我感到非常不安和困擾。因此，我對戴手錶沒有太大的興趣，更喜歡在運動時保持手腕的自由。最後，感謝同學的分享，讓我有機會表達自己的想法。

**Reply by wanderer657** [reply]
我懂，對喜歡球類運動的人來說，戴手錶可能不便且安全風險高。你的個人經歷生動展示了戴手錶可能引發的意外傷害。同時，擔心手錶遺失或被偷也是現實問題，增加了使用手錶的不便和焦慮。因此，不戴手錶能確保手腕自由，避免安全隱患。感謝分享，讓我更了解手錶使用的考量和困擾。

**Comment by YifenLin** [comment]
我本來是會習慣戴手錶的人，不過上大學之後加了球隊，打球常常需要把錶拿下來，後來就漸漸的不戴錶了，甚至考試會忘記帶🤧🤧

**Reply by 155Julia** [reply]
我也是！！

**Comment by wanderer657** [comment]
我記得個人小時候也很常帶手錶，當時受限於沒有智慧型裝置的環境，無法隨時查看時間確認作息，但當國中時人手一機逐漸普及後，手錶成為我認為有些多餘的事物，多年沒有戴上，漸漸就放著生灰了，但近期我覺得手錶也是穿搭的一環，其可以點綴手腕處的層次感，提高人們身上的細節多元性。

**Comment by SusuTima** [comment]
小時候到高中以前都有使用手錶的習慣，升大學手機使用頻繁後，手錶的存在就比較多餘了。現在段考的時候依舊不會使用，只會帶一個小鬧鐘（不會響）進去。

**Comment by sunset513** [comment]
我覺得在考試的時候一定要配戴手錶，我曾經有考試考到一半突然很緊張，因為我不確定剩下的時間還有多少，教室裡也沒有時鐘。我只能趕快的寫完，確保能夠交出一張寫完的考券。從此之後，手錶成為了我的必定配戴的物件。

**Comment by OG-Matcha** [comment]
我以前是有佩戴手錶的習慣，因為在國中、高中的時候平常沒法用手機，但現在到大學後基本上手機隨時都在身邊，只有在考試的時候才會用到，時間久了，我也就越來越少戴手錶的習慣了。

**Reply by wanderer657** [reply]
手機的多功能取代了手錶的需求，而且手錶的風格和品牌也影響了使用者的選擇。手錶作為配飾，如果不符合個人品味或身份象徵，可能會降低興趣。我自己以前戴手錶，但隨著手機普及，不再需要。另外，運動時戴手錶可能造成受傷，所以我不再使用。

**Comment by KH1126** [comment]
我只有在學期中佩戴手錶的習慣，
但每次一到放假，因為不太需要隨時關注時間，
手錶就會被我遺落在角落。

**Comment by jim1743** [comment]
我以前有佩戴手錶過，但因為手錶的錶帶很常斷掉，而且在高中上課時也有時鐘可以看時間，因而漸漸地放棄帶它，現在幾乎都是看手機，考試也是不戴手錶，憑感覺判斷時間到哪。

**Comment by lyykey** [comment]
我有一個習慣不知道大家有沒有，春秋冬會帶手錶的機率比夏高太多，因為夏天戴手錶錶帶會很容易悶濕，真的不舒服。
另外，目前我偏愛戴 Tissot、Longines，不知道大家喜歡什麼。

**Reply by 20050201Shawn** [reply]
同意，夏天錶帶跟手腕之間很容易卡汗然後滋生細菌，不舒服。另外我個人只戴過夜市廉價機械錶跟swatch，帶swatch兩年了覺得還不錯，沒什麼功能上的問題，設計也蠻喜歡的。

**Comment by nelson910402** [comment]
我是偶爾戴機械錶或電子錶，主要是看方便，偶爾會配合傳搭。電子錶其實很推薦小米手環，雖然比不上apple watch之類的品牌，但基本常用的功能不會少

**Comment by EPHMR0524** [comment]
手錶我覺得是在考試時一定要帶的，但是在我大學以前我都是不戴手錶出門的，是升上大學後買一支有附帶一卡通功能的手錶才開始每天帶的，這樣出門買東西很方便，不用掏錢包出來。

**Comment by 20050201Shawn** [comment]
我有戴手錶的習慣，雖然手機很方便，但在通勤(比如騎腳踏車)的時候看手錶確認時間比拿出手機確認時間比較安全，動作也很方便，如果是看手機的話還要把手機拿出口袋再放回口袋。但天氣冷時若穿比較保暖，袖子還要拉下來才能看時間有點麻煩。

**Comment by woohsin** [comment]
其實每個人對於戴手錶的習慣都有所不同。隨著手機的普及和功能的提升，許多人確實會選擇使用手機來查看時間，而不再需要戴手錶。因此，現在有戴手錶的習慣的人數可能相對減少了一些。

**Comment by happy246810** [comment]
在小學，我有戴手錶的習慣，但因為我是一個很粗心的人，我只要把手錶拔下來我就會忘記我放在哪裡🤣。所以時間一長，我就漸漸不戴手錶了，改用手機看時間。我只有在考試的時候會戴手錶，因為沒辦法帶手機進考場🤣。但上大學後，發現手錶真的好重要，因為要去趕課😭。

**Comment by YANRONG0123** [comment]
由於考試時避免造成誤會，我都會把指針式手錶「放」在桌上，而不會戴在手上，因為戴著會覺得手腕會卡卡的，有點難做中等幅度的位移，而其他類型的高科技手錶因為經費緣故，所以交給其他人回答。

**Comment by yun1007** [comment]
之前會戴，但後來家人求了個手鍊要我們帶著不能取下來，顯得左手東西太多，就不戴了。

**Comment by 110101501** [comment]
自己為了看時間所以一直都有戴手錶的習慣。以前曾經是戴機械錶，自從買了運動手環後就一直戴手環了。除了有看時間的功能外還能看訊息設鬧鐘，功能算是齊全好用。

**Comment by 155Julia** [comment]
自從國小到高中，我一直有戴手錶的習慣。然而有一天上大學沒多久，手錶壞了，從那以後我就再也沒戴過。因為大學的考試大部分都有提供時間，而即使沒有，那些考試都是能夠在時間內完成。因此，我漸漸地就習慣了沒有手錶的生活，要我現在帶手錶反而覺得怪怪的不舒服。

**Comment by wanka2030** [comment]
如果有看時間的需求大部分的人應該會選擇用手機來看，不僅是因為它的便利性以及手機不離身的原因，更多的是手機能夠在你的身邊。但在考試的時候手錶卻是相當重要的，只要沒有時鐘的考試總會特別緊張、特別焦慮，所以如果能培養戴手錶的習慣還是相當重要。

**Comment by Mitochondrionjessica** [comment]
一定要! 國小低年級養成戴手錶的習慣，到了大學還是持續帶著(當然是從可愛的卡通錶換到運動錶再到智慧手錶)。不同場合也會配戴不同功能的手錶，像是運動時會配戴運動錶紀錄自己的運動軌跡，參加正式活動正裝打扮的時候會配戴父母送我的手錶，象徵也紀念每一個重要的場合(或許父母不能出席，但看到他們送的錶繫在手腕上就覺得放心)。所以對我來說每一隻錶都是珍貴無比的，記錄著我生命中分分秒秒的回憶!

**Comment by willychiuchiu** [comment]
小時候看同學戴手錶覺得很帥就也跟著戴一段時間，但因為自己流汗的時候容易起疹子，每次只要到了夏天或是運動容易流汗的時候，戴手錶的地方都會特別癢而且手錶也容易有卡髒污，所以後來就都把手錶放在包包裡，只有考試的時候會拿出來

**Comment by tzurou** [comment]
我自己沒有酨手錶的習慣，夏天天氣熱的時候帶著很不舒服，手錶也容易髒很難弄乾淨，想看時間手機點一下就知道，只有考試的時候才會把塵封已久的手錶拿出來，不戴手錶的缺點大概就是每次考試前都要多檢查手錶有沒有帶

**Comment by yxin214** [comment]
記得國小時第一份生日禮物就是手錶，用了幾年後跌倒不小心摔壞還難過了好幾天。現在戴手錶的人越來越少，包括我自己，因為用手機看時間太方便了，也不會有戴手錶的異物感。但我想手錶還是有存在的必要，畢竟許多考試都不開放帶手機入場，或許懷錶能成為未來的趨勢？

**Comment by LinChloe** [comment]
因為考試需要看手錶，但如果我平常不帶的話，考試當天多半會忘記帶出來，所以就讓它一直待在手上了。

**Comment by cassy0201** [comment]
我覺得這跟習慣有關，我自己小時候帶習慣後，長大就會一直戴著，連洗澡都不會拆下來

**Comment by 710388** [comment]
肯定是有的，戴手錶看時間才方便，懶得拿手機出來，而且手錶還可以有附加功能像是檢測心跳

**Comment by yubo0oo** [comment]
只有考試或是不方便使用手機的場合會帶著

**Comment by MingWei174** [comment]
我自己習慣戴小米手環(大約5年)，但到了大型考試要戴傳統式手錶時無法馬上知道準確時間時，我會有一種不安的感覺，戴電子錶習慣了真的無法再回去戴指針錶....

**Comment by hunteryang0108** [comment]
我在國中以前是習慣每天配戴手錶的，上高中後因為能隨身攜帶手機，漸漸地就沒有戴手錶的習慣了，拿出手機看時間取代了手錶的作用。不過在一些特殊情況下還是會把手錶拿出來戴，例如無法使用手機時：考試，因此手錶的存在還是有作用的。

**Comment by Nana1218** [comment]
在高中前我是會每天戴手表的，但從上高中開始就沒有配戴了，我覺得最大的原因是國中上學時手機會被收走統一保管，但高中後學校就不再收手機，而手機不離身的結果就是原本手表的功能被手機取代了，自然而然就不需要再繼續在日常生活中佩戴手表。

**Comment by kingsophy** [comment]
我有戴手錶的習慣，只要是出門都會戴。可能是受到國中高中的影響，能夠隨時知道現在幾點幾分對我來說很重要，因為想要早一點下課或吃飯，可以讓我有一種安全感存在。

**Comment by ds0w0** [comment]
對我來說，手錶戴著確實有點不太舒適，尤其是當我參加考試或做其他活動時。像我平時會把手錶放在口袋裡，只有在考試、赴約等必要的時候才會戴上。對於喜歡打音遊、彈琴和打爵士鼓的人來說，手錶可能會有點彆扭。
